doc/source/whatsnew/v0.21.0.txt

```diff
@@ -34,6 +34,7 @@ Other Enhancements
 Backwards incompatible API changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- The ``DataFrame.style.to_excel()`` styled Excel export functionality now requires an external dependency, `cssdecl <http://pypi.python.org/pypi/cssdecl>`_.
```

Comment:
doesn't this add export to xlxswriter as compared to 0.20.1? if so pls add *another* note in Other Enhancements section. (maybe also need to update the io.rst docs?)